app/app.js

```diff
@@ -9,7 +9,7 @@ angular.module('myApp', ['ngRoute', 'myApp.version', 'ui.router', 'ui.bootstrap'
     "$rootScope", "$state", "$stateParams", "DataService", function ($rootScope, $state, $stateParams, DataService) {
       $rootScope.$state = $state;
       // return $rootScope.$stateParams = $stateParams;
-      DataService.createApplication({review: {}}).then(function (result) {
+      DataService.createApplication({review: {}},{applicantInfo: {}}).then(function (result) {
```

Comment:
This should be:
`DataService.createApplication({review: {}, applicantInfo: {}}).then(function (result) {`